scripts/configure-dh.sh

```diff
@@ -131,6 +131,30 @@ fi
 
 echo "OK"
 
+# Reads Model URL for lightspeed plugin
+# Optional: If no URL is entered, lightspeed plugin will not be configured
+if [ -z "${LIGHTSPEED_MODEL_URL}" ]; then
+    read -p "Enter your model URL for lightspeed (Optional): " LIGHTSPEED_MODEL_URL
+
+    # Reads API token for lightspeed plugin
+    # Optional: If no token is entered, lightspeed plugin will not use authenticated communication
+    if [ -z "${LIGHTSPEED_API_TOKEN}" ]; then
+        read -p "Enter API token for lightspeed (Optional): " LIGHTSPEED_API_TOKEN
+    fi
+    echo "OK"
+fi
+if [ ! -z "${LIGHTSPEED_MODEL_URL}" ]; then
+    # make sure the target url ends in /v1
+    if ! [[ $LIGHTSPEED_MODEL_URL =~ \/v1$ ]]; then
+        LIGHTSPEED_MODEL_URL="$LIGHTSPEED_MODEL_URL/v1"
+    fi
+
+    plugin_sha=$(npm view @janus-idp/backstage-plugin-lightspeed dist.integrity)
+
+    cp -f $BASE_DIR/resources/lightspeed-plugins.yaml $BASE_DIR/dynamic-plugins/
```

Comment:
If we move the plugins to `/dynamic-plugins` do we need the cp line here?

Reply:
Instead of copying into `/dynamic-plugins` we could add a list of file paths to extend the base list from `/dynamic-plugins`, e.g. if `LIGHTSPEED_MODEL_URL` is defined, add `$BASE_DIR/optional-plugins/lightspeed-plugins.yaml` file to the extended list to iterate over.

Reply:
moved to the `optional-plugins` approach, it does look better